modules/cnv_storage_wizard_fields_web.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//cnv_storage_wizard_fields_web.adoc
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-storage-wizard-fields-web]]
+== Storage fields
+|===
+|Name | Description
+
+|Create Disk
+|Create a new disk for the virtual machine.
+
+|Attach Disk
+|Select an existing disk, from a list of available PVCs, to attach to the virtual machine. 
+
+|DISK NAME
+|Name of the disk.
+
+|SIZE (GB)
+|Size, in GB, of the disk. 
+
+|STORAGE CLASS
+|Name of the underlying `StorageClass`. 
+
+|Bootable Disk
+|List of available disks from which the virtual machine will boot.
```

Comment:
@jelkosz Please confirm we need to be more specific in [1, 2, 3, 4] or elsewhere. That rootdisk is not only default bootable disk for Container and URL sources, but it's only available option which can't be changed or removed. For PXE one can change disks, but there is always must be one (bootable).

In other words default doesn't mean "(un)changeable". 
And rootdisk is not the same as a bootable disk. [3, 4].

@aburdenthehand  Also I would move some wording of lines [1, 2] and a note "One can't remove a rootdisk" into xref:cnv_storage_wizard_fields_web.adoc#cnv-storage-wizard-fields-web[*Storage*]   

[1] https://github.com/openshift/openshift-docs/pull/13489/files#diff-9ea9eed2e360d410a865899f28c1925bR22
[2] https://github.com/openshift/openshift-docs/pull/13489/files#diff-5b75dac4831be9f2ca87957b91d4aa0aR21
[3] https://github.com/openshift/openshift-docs/pull/13489/files#diff-9ea9eed2e360d410a865899f28c1925bR23
[4] https://github.com/openshift/openshift-docs/pull/13489/files#diff-5b75dac4831be9f2ca87957b91d4aa0aR22

Reply:
> That rootdisk is not only default bootable disk for Container and URL sources, but it's only available option which can't be changed or removed. 

indeed, this is the case

> For PXE one can change disks, but there is always must be one (bootable).

For PXE you don't need to have any disks. But if you do, you have to pick one which is the bootable one.

>